.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. Ensure that this is the correct path for your virtual environment. If your virtual environment is named differently, you should adjust this entry accordingly.